src/main/config/spotbugs/spotbugs-exclude.xml

```diff
@@ -0,0 +1,14 @@
+<FindBugsFilter>
+  <Match>
+    <Class name="~org\.apache\.phoenix\.coprocessor\.generated\..*" />
+  </Match>
+  <Match>
+    <BugPattern name="NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE" />
```

Comment:
Good to comment what we are excluding here, just a short text is fine

Reply:
Done. 
Also removed the synamic SQL Statement exlcusion, and opened PHOENIX-6560 as a follow-up